python/pyproject.toml

```diff
@@ -0,0 +1,132 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0", "wheel"]
+
+[project]
+authors = [
+  {name = "Cucumber Limited", email = "cukes@googlegroups.com"}
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: POSIX",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: MacOS :: MacOS X",
+  "Topic :: Software Development :: Testing",
+  "Topic :: Utilities",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13"
+]
+dependencies = []
+description = "Cucumber Messages is a message protocol for representing results and other information from Cucumber. "
+license = {text = "MIT"}
+maintainers = [
+  {name = "Konstantin Goloveshko", email = "kostya.goloveshko@gmail.com"}
+]
+name = "cucumber-messages"
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.9"
+urls = {Repository = "https://github.com/cucumber/messages"}
+version = "0.1.0"
+
+[project.optional-dependencies]
+test = [
+  # local
+  "cucumber-messages[test-coverage]",
+  # external; Must be in sync with [tool.tox]
+  "mypy",
+  "pre-commit",
+  "tox>=4.2"
+]
+test-coverage = [
+  "coverage",
+  "GitPython",
+  "packaging",
+  "pytest"
+]
+
+[tool.black]
+# Don't include autogenerated file
+force-exclude = ".*\\/src\\/cucumber_messages\\/_messages\\.py"
```

Comment:
Is this line required now? This file **is** being checked in

Reply:
Yes, it's needed for other not generated python files